src/main/resources/swagger/cromwell.yaml

```diff
@@ -507,6 +507,79 @@ paths:
       security:
         - google_oauth:
             - openid
+  '/workflows/{version}/{id}/call-caching':
+    post:
+      summary: Alter call cache result reuse settings for all calls within the specified workflow.
+      parameters:
+        - name: version
+          description: API Version
+          required: true
+          type: string
+          in: path
+          default: v1
+        - name: id
+          description: Workflow ID
+          required: true
+          type: string
+          in: path
+        - name: allow
+          description: Whether to allow call cache result reuse for all calls within the specified workflow.
+          required: true
+          type: boolean
+          in: query
+      tags:
+        - Workflows
+      responses:
+        '200':
+          description: Successful Request
+          schema:
+            $ref: '#/definitions/CallCachingResponse'
+        '400':
+          description: Validation error
+        '500':
+          description: Internal Error
+      security:
+        - google_oauth:
+            - openid
+  '/workflows/{version}/{id}/call-caching/{callFqn}':
```

Comment:
I collapsed the Spray endpoints per @cjllanwarne's suggestion, but unfortunately optional path parameters are not currently allowed in Swagger:

https://github.com/swagger-api/swagger-spec/issues/93

So while this Swagger YAML and the README.md document two endpoints, in reality Cromwell only implements one endpoint which can handle both the whole workflow and individual call use cases.  If a future version of Swagger adds support for optional path parameters we can change these docs to more closely match the underlying reality.